lib/src/chord_parser.dart

```diff
@@ -13,9 +13,67 @@ class ChordProcessor {
     required String text,
     required TextStyle lyricsStyle,
     required chordStyle,
+    required widgetPadding,
```

Comment:
```suggestion
    widgetPadding = 0,
```
Set it to be `0` by default.